config/jobs/kubernetes-sigs/kube-storage-version-migrator/presubmits.yaml

```diff
@@ -0,0 +1,15 @@
+presubmits:
+  kubernetes-sigs/kube-storage-version-migrator:
+  - name: pull-migrator-test
+    decorate: true
+    always_run: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20180912-7a027de0c-master
+        args:
```

Comment:
I think you can drop all args section

Reply:
yeah, after dropping the `args:` section this should be good to go

Reply:
Done.

Reply:
I don't think it pushed?